docs/geedocs/docsrc/answer/geeServerConfigAndSecurity/configureGeeServer5.1.0_SSL_HTTPS.rst

```diff
@@ -172,16 +172,16 @@ Configure GEE Server 5.1.0 for SSL/HTTPS
             Use of the ``SSLRequireSSL`` directive prevents all HTTP
             requests that do not use SSL, thereby protecting your data
             from all but HTTPS requests.
-            See `Apache HTTP Server Version 2.2 Documentation
-            <http://httpd.apache.org/docs/2.2/mod/mod_ssl.html#sslrequiressl>`_
+            See `Apache HTTP Server Version 2.4 Documentation
```

Comment:
This document refer to GEE server 5.1.0. I see two options:
- Make it version neutral: rename the the .rst to remove 5.1.0 from it and edit line 36 to remove the version number. Also there is a link to this document from "Manage virtual hosts" document that need to be updated in two places.
- Keep it as 5.1.0 but undo the doc changes since 5.1.0 does not use Apache 2.4.

My preference is to go with first option.

Reply:
I like your first option better, too, @tst-nfarah . The documentation here applies well beyond version 5.1.0. Let's go with that.